odoo/addons/base/models/ir_ui_view.py

```diff
@@ -569,6 +569,12 @@ def unlink(self):
             self.inherit_children_ids.unlink()
         return super(View, self).unlink()
 
+    def _update_field_translations(self, fname, translations, digest=None):
+        res = super()._update_field_translations(fname, translations, digest)
+        if fname == 'arch_db' and 'install_filename' not in self._context:
+            self.write({'arch_updated': True})
```

Comment:
Does that mean that changing a translation will make the view considered as arch_updated?
If so, that's not supposed to be the case, as having the arch_updated has some impact we would like to minimize.
And the arch is not really updated, only it's translation is.
For the website part, that's mainly about it preventing the theme views to not be updated in such a case on the related website.

Reply:
I write 'arch_db' only to notifiy the arch value is updated
so the web client can reload it after translating

I think in master it is done by clear_cache() in ir.translation, which seems a little bit overkill
I have fully understand how website works, but I can do the same thing (clear orm cache) in this override. 

Could you tell me if there is any other way can do the same thing?